test/features/caseView.feature

```diff
@@ -30,15 +44,3 @@ Feature: Set of scenarios testing case view functionality
       | Text Field 2        |
       | Text Field 3        |
       | Text Field 4        |
-
-  @functional
```

Comment:
Are we completely deleting these Scenarios for some reason? or was this a mistake?